### PR TITLE
add logo and CHRC to directory/directory.json

### PR DESCRIPTION
**Pull request by jonasess**
uploaded logo of CHRC
and edited the directory.json

### CODE REVIEW

**Comment by jonasess** [comment]
would please tell me what left? or what's the mistake?

File: directory/directory.js [hunk 1]
```diff
@@ -452,6 +451,10 @@ directory.addPair({
   baseBuying: ['XLM', 'native'],
   counterSelling: ['BTC', 'golix.io'],
 });
+directory.addPair({
+  baseBuying: ['XLM', 'native'],
+  counterSelling: ['BTC', 'charnatoken.top'],
```
Comment (irisli) [inline directory/directory.js]: CHRC is not a BTC. It should be under "Pairs involving XLM"

File: directory/directory.js [hunk 2]
```diff
@@ -370,17 +370,16 @@ directory.addAsset('cryptomover.com', {
...
 });
-
 directory.addAnchor({
-  domain: 'thefutbolcoin.io',
-  website: 'https://thefutbolcoin.io/',
-  logo: 'thefutbolcoin.io',
-  color: '#4b914e',
-  displayName: 'TFC',
+  domain: 'charnetoken.top',
+  website: 'http://charnatoken.top/',
```
Comment (irisli) [inline directory/directory.js]: Needs to be https

File: directory/directory.js [hunk 3]
```diff
@@ -370,17 +370,16 @@ directory.addAsset('cryptomover.com', {
   code: 'HKDC',
   issuer: 'GA4BYMUO5D7OLGVJWZ2D5FCWU7SB63FNZ4QUU574SMNA6ELK5TZD3SO3',
 });
-
 directory.addAnchor({
-  domain: 'thefutbolcoin.io',
-  website: 'https://thefutbolcoin.io/',
-  logo: 'thefutbolcoin.io',
```
Comment (irisli) [inline directory/directory.js]: You're not supposed to delete other stuff

**Comment by irisli** [comment]
You also need to rebase against master

File: directory/directory.json [hunk 4]
```diff
@@ -228,15 +228,17 @@
...
+
+	"charnatoken.top": {
```
Comment (irisli) [inline directory/directory.json]: This is autogenerated by ./checkBuild.sh. You shouldn't be manually adding this

File: directory/directory.js (hunk 1 above)
Comment (jonasess) [reply directory/directory.js]: okay . but we want to exchange CHRC with btc is that possible?

File: directory/directory.json (hunk 4 above)
Comment (jonasess) [reply directory/directory.json]: okay . so i only have to change website to https and then maunally edit only on these files
directory.js directory.json  logos.json

File: directory/directory.js (hunk 2 above)
Comment (jonasess) [reply directory/directory.js]: okay working on that

**Pull request closed without merging**